src/scikit_package/app.py

```diff
@@ -2,22 +2,31 @@
 from argparse import ArgumentParser
 
 
-def create():
-    run_cookiecutter()
+def create(package_type):
+    if package_type == "workspace":
+        run_cookiecutter(
+            "https://github.com/Billingegroup/scikit-package-workspace"
+        )
+    elif package_type == "system":
+        run_cookiecutter(
+            "https://github.com/Billingegroup/scikit-package-system"
+        )
+    elif package_type == "public":
+        run_cookiecutter("https://github.com/Billingegroup/scikit-package")
 
 
 def update():
     # FIXME: Implement the update command.
     # As of now it does the same as the create command.
-    run_cookiecutter()
+    run_cookiecutter("https://github.com/Billingegroup/scikit-package")
```

Comment:
Let's pull this out to a global variable at the top of the file, also maybe the URLs above.